WordPress-Core/ruleset.xml

```diff
@@ -389,23 +356,64 @@
 	<!-- Rule: The short ternary operator must not be used. -->
 	<rule ref="Universal.Operators.DisallowShortTernary"/>
 
+
 	<!--
 	#############################################################################
-	Handbook: PHP - Yoda Conditions.
-	Ref: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions
+	Handbook: PHP - (No) Error Control Operator @.
+	Ref: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#error-control-operator
+	#############################################################################
+	-->
+	<!-- Covers rule: This operator is often used lazily instead of doing proper error checking.
+		 Its use is highly discouraged. -->
+	<rule ref="WordPress.PHP.NoSilencedErrors"/>
+
+
+	<!--
+	#############################################################################
+	Handbook: PHP - Database Queries.
+	Ref: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#database-queries
+	#############################################################################
+	-->
+	<!-- Covers rule: Avoid touching the database directly. -->
+	<rule ref="WordPress.DB.RestrictedFunctions"/>
+	<rule ref="WordPress.DB.RestrictedClasses"/>
+
+
+	<!--
+	#############################################################################
+	Handbook: PHP - Formatting SQL statements.
+	Ref: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#formatting-sql-statements
+	#############################################################################
+	-->
+	<!-- Rule: Always capitalize the SQL parts of the statement like UPDATE or WHERE.
+		 https://github.com/WordPress/WordPress-Coding-Standards/issues/639 -->
+
+	<!-- Rule: Functions that update the database should expect their parameters to lack
+		 SQL slash escaping when passed.
+		 https://github.com/WordPress/WordPress-Coding-Standards/issues/640 -->
+
+	<!-- Rule: in $wpdb->prepare - %s is used for string placeholders and %d is used for integer
+		 placeholders. Note that they are not 'quoted'! -->
+	<rule ref="WordPress.DB.PreparedSQLPlaceholders"/>
+
+	<!-- Covers rule:  $wpdb->prepare()... The benefit of this is that we don't have to remember
+		 to manually use esc_sql(), and also that it is easy to see at a glance whether something
+		 has been escaped or not, because it happens right when the query happens. -->
```

Comment:
```suggestion
	<!-- Covers rule:  Escaping should be done as close to the time of the query
		 as possible, preferably by using $wpdb->prepare(). -->
```

Should maybe also be moved up to above the previous rule to stay in line with the handbook.